sonarcloud/README.md

```diff
@@ -0,0 +1,32 @@
+# OPA Test Coverage to SonarCloud Format
+
+Example code to transform the OPA test coverage JSON report to
+the [SonarCloud](https://docs.sonarqube.org/9.8/analyzing-source-code/test-coverage/generic-test-data/) coverage report format.
```

Comment:
Should we perhaps use https://docs.sonarqube.org/latest/analyzing-source-code/test-coverage/generic-test-data/ 

i.e. latest, rather than a fixed version?